indico/modules/events/contributions/templates/management/_contribution_protection_message.html

```diff
@@ -0,0 +1,71 @@
+<div class="form-group" id="form-group-protected">
+    <div class="form-label form-block"></div>
+    <div class="form-field form-block">
+        {% set protection_parent = contrib.protection_parent %}
+
+        <div class="action-box message-only for-form public-protection-message danger">
+            <div class="section">
+                <div class="icon icon-unlocked"></div>
+                <div class="text">
+                    <div class="label">
+                        {% trans %}Public{% endtrans %}
+                    </div>
+                    <div>
+                        {% trans protection_parent_title=protection_parent.title %}
```

Comment:
you always want to trim whitespace inside trans (i.e. `trans -%}...{%- endtrans`)